Dockerfile.web

```diff
@@ -1,11 +1,11 @@
 FROM node:8.9.0 as node
 
-WORKDIR /work
+  WORKDIR /work
```

Comment:
Is this a convention?  The whitespace doesn't seem necessary but I'm not very familiar with Dockerfile conventions.

Reply:
Good catch! I thought it was suppose to be indented but after some reading, looks like it shouldn't .